rcon/extended_commands.py

```diff
@@ -218,6 +218,21 @@ def do_add_vip(self, name, steam_id_64):
         with invalidates(Rcon.get_vip_ids):
             return super().do_add_vip(steam_id_64, name)
 
+
+    @ttl_cache(ttl=60)
+    def remove_all_vips(self):
+        with invalidates(Rcon.get_vip_ids):
+            vips = self.get_vip_ids()
+            for vip in vips:
+                try:
+                    self.do_remove_vip(vip['steam_id_64'])
+                except ValueError:
```

Comment:
If there a failed command it will raise `CommandFailedError` after retrying once.
So here it's better to just catch this one, and `continue` instead. The manual reconnection won't help much here